rs_push

```diff
@@ -77,7 +77,7 @@ if [ ! $use_current_metadata ]; then
     echo ""
     echo "Please run this command to fix the version numbering ... "
     echo ""
-    echo "  sed -i -e \"/^version/s/[\\'\\\"][0-9.]*[\\'\\\"]/'${vers}'/\" */metadata.rb"
+    echo "  sed -i -e \"/^version/s/[\\'\\\"][0-9.]*[\\'\\\"]/'${version_template}'/\" */metadata.rb"
```

Comment:
Understand why this was done.  However, line 63 should be updated as well to handle development branches.